bridges/scalajs-1/src/main/scala/bloop/scalajs/jsenv/NodeJsEnv.scala

```diff
@@ -96,7 +100,7 @@ object NodeJSConfig {
  *
  * Adapted from `scala-js/nodejs-env/src/main/scala/org/scalajs/jsenv/nodejs/NodeJSEnv.scala`.
  */
-final class NodeJSEnv(logger: Logger, config: NodeJSConfig) extends JSEnv {
+final class NodeJSEnv(logger: Logger, config: NodeJSConfig, ioScheduler: Scheduler) extends JSEnv {
```

Comment:
Sorry to comment about it again, but it seems that we are anyway using  `engine.ExecutionContext.ioScheduler`, so maybe we can use that? Maybe adding it to params was not needed?

Reply:
oh. I missed that jsBridge depends on frontend. Changed.